Syntaxes/debian-changelog.sublime-syntax

```diff
@@ -9,30 +9,84 @@ scope: source.debian.changelog
 
 contexts:
   main:
-    - include: changelog-header
-    - include: changelog-footer
-    - include: content-close-bug
-    - include: content-code-span
+    - include: header
+
+  header:
+    - match: ^[\w-]+
+      scope: entity.name.class.debian.changelog
+      push: [body, header-annotations]
 
-  changelog-header:
-    - match: (.+) [(](.+)[)] (.+); (.+)
+  header-annotations:
+    - match: \(
+      scope: punctuation.section.group.begin.debian.changelog
+      push:
+        - meta_scope: meta.version-number.debian.changelog
+                      meta.parens.debian.changelog
+        - include: debian-common.sublime-syntax#semantic-version
+        - match: \)
```

Comment:
There is some weird highlighting going on in version numbers:

![Screenshot from 2021-01-18 08-36-57](https://user-images.githubusercontent.com/2029239/104885496-6d4a7880-5968-11eb-882c-21f7fbec0f84.png)